src/main/java/cubicchunks/world/column/BlankColumn.java

```diff
@@ -59,6 +60,10 @@ public Cube removeCube(int cubeY) {
 		return blankCube;
 	}
 
+	public void addCube(Cube cube) {
+		throw new UnsupportedOperationException("Server must have sent Cubes before Columns");
```

Comment:
Turns out there already is a check in ClientHandler... (removing the one in BlackColumn)